web/src/components/tasks/RaffleBingoSteps.ts

```diff
@@ -0,0 +1,43 @@
+import { fetchTaskByFilename } from "@/lib/async-content-fetchers/fetchTaskByFilename";
+import { Task } from "@/lib/types/types";
+import { arrayOfTaskAgencies } from "@businessnjgovnavigator/shared/taskAgency";
+
+type RaffleBingoStepLabels = "Get an ID Number" | "From Your NJ Municipality";
+
+export const RaffleBingoSteps: {
+  stepLabel: RaffleBingoStepLabels;
+  fileName: string;
+  agencyId: string;
+}[] = [
+  {
+    stepLabel: "Get an ID Number",
+    fileName: "raffle-license-step-1",
+    agencyId: "games-of-chance-control-commission",
+  },
+  { stepLabel: "From Your NJ Municipality", fileName: "raffle-license-step-2", agencyId: "nj-municipality" },
```

Comment:
[pebble] Things like `stepLabel` and `agencyId` should probably be content driven. I suggest moving these values into the CMS. Then the 2 filenames representing the steps could be stored in an array.

In `RaffleBingoPaginator`, instances where you're referencing `RaffleBingoSteps` to get this content could look at the current step.